scripts/cluster-install/clean-openstack.py

```diff
@@ -0,0 +1,32 @@
+import yaml
```

Comment:
If you put a '#! /usr/bin/env python' here then those of us using posixy systems will probably be able to just run this with ./install.py <args> rather than having to type 'python install.py <args>'

Reply:
(this was meant for install.py, but the same applies here)

Reply:
Done, thanks.